### DISCUSSION TITLE
Please Default to Windows Terminal

### DISCUSSION THREAD

**Discussion by sharpninja** [PowerShell/PowerShell#14817] [General]
When PS is installed on a system with Windows Terminal, please default to that console.

**Comment by SteveL-MSFT** [comment]
I believe this is something the Windows Terminal team can control as they also own the Windows console and could change the default conhost to Windows Terminal if installed.  I would suggest opening an issue in their repo.  https://github.com/Microsoft/Terminal

**Reply by zadjii-msft** [reply]
For future readers - this is already tracked in microsoft/terminal#492
